### PR TITLE
Generate activate.ps1 on all systems

### PR DESCRIPTION
**Pull request by karazijal**

### CODE REVIEW

**Comment by karazijal** [comment]
Issue #1209

**Comment by gaborbernat** [comment]
@karazijal thanks for this; there's a slight concern here now: we add this to all environments but will it work if we don't use the platform dependent line ending?  maybe before copy, we should normalize to the platform line separator 🤔 can you check and test this?

File: tests/test_cmdline.py [hunk 1]
```diff
@@ -9,8 +10,19 @@
 
 
 def test_commandline_basic(tmpdir):
-    """Simple command line usage should work"""
-    subprocess.check_call([sys.executable, VIRTUALENV_SCRIPT, str(tmpdir.join("venv"))])
+    """Simple command line usage should work and files should be generated"""
+    venv_dir = str(tmpdir.join("venv"))
+    venv_bin_dir = os.path.join(venv_dir, "bin")
```
Comment (gaborbernat) [inline tests/test_cmdline.py]: note this is not always bin, but is platform dependent :+1:

**Comment by ferdinandvanwyk** [comment]
Helping out on this as part Bloomberg PyPA sprint

**Comment by gaborbernat** [comment]
closing in favor if #1229

**Pull request closed without merging**